src/promptflow/promptflow/_cli/_utils.py

```diff
@@ -389,11 +389,14 @@ def get_secret_input(prompt, mask="*"):
     - Ignore control characters and print warning message.
     """
     if not isinstance(prompt, str):
-        raise TypeError(f"prompt must be a str, not ${type(prompt).__name__}")
+        e = TypeError(f"prompt must be a str, not ${type(prompt).__name__}")
+        raise UserErrorException(message_format=str(e)) from e
     if not isinstance(mask, str):
-        raise TypeError(f"mask argument must be a one-character str, not ${type(mask).__name__}")
+        e = TypeError(f"mask argument must be a one-character str, not ${type(mask).__name__}")
+        raise UserErrorException(message_format=str(e)) from e
     if len(mask) != 1:
-        raise ValueError("mask argument must be a one-character str")
+        e = ValueError("mask argument must be a one-character str")
+        raise UserErrorException(message_format=str(e)) from e
```

Comment:
why do we need the inner `ValueError`, feel like we can simply change the original `TypeError` to `UserErrorException`?

Reply:
Because it can display the most primitive error types:
![image](https://github.com/microsoft/promptflow/assets/23182548/11c08ffc-6423-439f-b066-2d313d745945)